src/components/event/EventDetails.js

```diff
@@ -414,79 +420,63 @@ class EventDetails extends Component {
-						<View style = {{ flex: 0.45 }}>
-							<Button
-								title = "Manual check-in"
-								onPress = { () => this.setState({ pickerVisible: true }) }
-							/>
-						</View>
-					</View>
-					<View style = {{ flexDirection: "row", justifyContent: "space-evenly", alignItems: "center", position: "absolute", bottom: dimension.height * 0.032, width: "100%" }}>
-						<View style = {{ flex: 0.45 }}>
-							<Button
-								title = "Edit event"
-								onPress = { () => {
-									this.props.startTimeChanged(this.prepend0(this.props.startTime));
-									this.props.endTimeChanged(this.prepend0(this.props.endTime));
-									this.props.goToCreateEventFromEdit(this.props.screen);
-								} } />
-						</View>
-						<View style = {{ flex: 0.45 }}>
-							<Button
-								title = "Delete event"
-								onPress = { () => Alert.alert("Confirmation", "Are you sure you want to delete", [
-									{
-										text: "Confirm",
-										onPress: () => this.deleteButton()
-									},
-									{
-										text: "Cancel"
-									}
-								]) }
-							/>
-						</View>
-					</View>
-				</View>
-			);
-		else
-			return (
-				<View>
-					{ this.limitRSVP(this.props.date) && <View style = {{ flexDirection: "row", justifyContent: "space-evenly", alignItems: "center", position: "absolute", bottom: dimension.height * 0.032, width: "100%" }}>
-						<View style = {{ flex: 0.45 }}>
-							<Button
-								title = "Check in"
-								onPress = { () => { this.setState({ modalVisible: true }) } }
-							/>
-						</View>
-						<View style = {{ flex: 0.45 }}>
-							<Button
-								title = "RSVP"
-								onPress = { () => {	this.renderRSVP() } }
-							/>
-						</View>
-					</View> }
-					{ !this.limitRSVP(this.props.date) && <View style = {{ flexDirection: "row", justifyContent: "space-evenly", alignItems: "center", position: "absolute", bottom: dimension.height * 0.032, width: "100%" }}>
-						<View style = {{ flex: 0.3 }}></View>
-						<View style = {{ flex: 1 }}>
-							<Button
-								title = "Check in"
-								onPress = { () => { this.setState({ modalVisible: true })	} }
-							/>
-						</View>
-						<View style = {{ flex: 0.3 }}></View>
-					</View> }
-				</View>
-			);
+		let buttons = [];
+
+		if (this.props.privilege && this.props.privilege.board) {
+			buttons = <ButtonLayout>
+				<Button
+					title = "Open check-in"
+					onPress = { this.openCheckInButton.bind(this) }
+				/>
+				{ this.renderPickMembers() }
+				<Button
+					title = "Edit event"
+					onPress = { () => {
+						this.props.startTimeChanged(this.prepend0(this.props.startTime));
+						this.props.endTimeChanged(this.prepend0(this.props.endTime));
+						this.props.goToCreateEventFromEdit(this.props.screen);
+					} }
+				/>
+				<Button
+					title = "Delete event"
+					onPress = { () => Alert.alert("Confirmation", "Are you sure you want to delete", [
+						{
+							text: "Confirm",
+							onPress: () => this.deleteButton()
```

Comment:
Can you name that function something else like confirmDelete or something that makes it sound like it's a function and not a component